FCCee/CAD_beampipe/README.txt

```diff
@@ -0,0 +1,26 @@
+!-- aciarma 17.07.2023
+
+Engineered beam pipe model + copper cooling
+CAD design courtesy of F. Fransesini
+
+To display the beampipe:
+geoDisplay standalone_CADbp.xml
+
+To use in simulation, include in the main xml file:
```

Comment:
Do I understand correctly that both files should be included to have a 'complete' beampipe? Can you say a word why the beampipe is 'split' into two compact files?

Reply:
now only one .xml file for the beampipe